metricbeat/module/apache/apache.asciidoc

```diff
@@ -15,5 +15,21 @@ The Apache Module has currently one MetricSet which is called status. It fetches
 === Apache Status Metricset
 
 
-TODO
-* List available selectors
+The apache status metricset offers the following selectors:
+
+* uptime
+* cpu
+* connections
+* load
+* scoreboard
```

Comment:
Yes, especially now that the selectors got replaced by filters https://github.com/elastic/beats/pull/1384